packages/sales-report-tool/index.js

```diff
@@ -0,0 +1,107 @@
+const fastcsv = require("fast-csv");
+const fs = require("fs");
+const ws = fs.createWriteStream("data.csv");
+const fetch = require("cross-fetch");
+
+// Data formating
+const getFormattedDate = (date) => {
+  const theDate = new Date(date * 1000);
+  const dateString = theDate.toGMTString();
+  return dateString;
+};
+
+// Array for storing transactions
+const arrOfTxn = [];
+let total;
+// Minting wallet address
+const address = `SP2X0TZ59D5SZ8ACQ6YMCHHNR2ZN51Z32E2CJ173`;
+// Mint price
+const mintPrice = 100;
+
+const getAccounting = async () => {
+  await fetch(
+    `https://stacks-node-api.mainnet.stacks.co/extended/v1/address/${address}/transactions`
+  )
+    .then((response) => response.json())
+    .then((data) => {
+      total = data.total;
+    });
+  let indexOfLastTxn = total;
+  let index = 1;
+  let nftsSold = 0;
+
+  while (indexOfLastTxn > 0 && indexOfLastTxn <= total) {
+    await fetch(
+      `https://stacks-node-api.mainnet.stacks.co/extended/v1/address/${address}/transactions?offset=${
+        indexOfLastTxn - 20
+      }`
+    )
+      .then((response) => response.json())
+      .then((data) => {
+        data.results &&
+          data.results.length > 0 &&
+          data.results.map((txn) => {
```

Comment:
Could be simplified with
```suggestion
        data?.results?.map((txn) => {
```